src/store/Store.ts

```diff
@@ -165,6 +161,17 @@ const Store = (): StoreOptions<StoreState> => {
         );
         state.themeIconDot = payload;
       },
+      setScopeThemingChecked: function (state: any, payload: boolean) {
```

Comment:
Does this mean it will have impact to all diagrams under this hostname?

Reply:
No, this only affects the check status, but I realized this status should always be the same as the current diagram, I will fix it.